pandas/plotting/_matplotlib/timeseries.py

```diff
@@ -307,7 +307,7 @@ def _maybe_convert_index(ax, data):
 
 
 def _format_coord(freq, t, y):
-    return "t = {0}  y = {1:8f}".format(Period(ordinal=int(t), freq=freq), y)
+    return f"t = {Period(ordinal=int(t), freq=freq)}  y = {y:8f}"
```

Comment:
IMO there is too much packed in this f-string. Could you put the ``Period(..)`` part in a varable and call that in the f-string?